components/f4jumble/src/test_vectors.rs

```diff
@@ -1,9 +1,18 @@
+#[cfg(feature = "std")]
+extern crate std;
+#[cfg(feature = "std")]
+use std::vec;
+#[cfg(feature = "std")]
+use std::vec::Vec;
+
+#[cfg(feature = "std")]
 pub(crate) struct TestVector {
     pub(crate) normal: Vec<u8>,
     pub(crate) jumbled: Vec<u8>,
```

Comment:
We could maybe change these to slices with static lifetime, to allow the test vectors to be used in no_std.